spydra/src/main/java/com/spotify/spydra/submitter/api/PoolingSubmitter.java

```diff
@@ -56,8 +56,6 @@
  * For each slot, it's cluster lifetime is offset by Age // Limit. This has as affect that
  * the limit is exceeded by one cluster at a time which should become idle and thus collected before
  * the next horde of clients come in. The algorithm is implemented in {@link ClusterPlacement#createClusterPlacement}.
- * Example scenario's have been worked out in
- * <a href="https://docs.google.com/spreadsheets/d/1Sfw_yxNSYJjHYtiHGQQf1ZHnNTqc7nGyXGGts_EaupM">a spreadsheet.</a>
```

Comment:
Any reason to remove this? It's accurate isn't it?

Reply:
It does not contain the new formula.